trelloapp.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import os
+from flask import Flask, render_template
+import json
+from dotenv import load_dotenv  #to invoke .env file
+load_dotenv()
+
+trelloapp = Flask(__name__)
+
+#Module2 project #
+#APP_KEY = os.environ.get('APP_KEY')  # take env variable from local system envor
+#APP_TOKEN=os.environ.get('APP_TOKEN')
+
+APP_KEY = os.getenv("APP_KEY")  # take .env from dotenv
+APP_TOKEN = os.getenv("APP_TOKEN")
+
+board_id= '5ef1186ddea5ff1b03e085e6'
+list_id_ToDo = '5ef1186d26a8d939ea575069'
+list_id_Pending = '5ef1186d2fd57d026f03add0'
+list_id_Done = '5ef1186dcbda554f16c6d66f'
+
+def get_card():
```

Comment:
Guessing this method is unused as it relies on a magic card id

Reply:
that's correct. was using that as a proof of concept to test getting card details.